WooCommerce/Classes/POS/Utils/Font+WooCommercePOS.swift

```diff
@@ -3,4 +3,6 @@ import SwiftUI
 extension Font {
     static let posBody: Font = Font.system(size: UIFontMetrics.default.scaledValue(for: 24))
     static let posTitle: Font = Font.largeTitle
+    static let posModalBody: Font = Font.system(size: UIFontMetrics.default.scaledValue(for: 24))
```

Comment:
These are the same as `posBody` and `posTitle` TfaZ4LUkEwEGrxfnEFzvJj-fi-3385_18076

We have a task to clearly define all the fonts from specification and maybe we should do it sooner rather than later. For this PR I suggest using the existing font definitions.